python/vowpalwabbit/pyvw.py

```diff
@@ -1270,6 +1277,69 @@ def __init__(self, action: int, score: float):
         self.score = score
 
 
+class CCBSlotOutcome:
+    def __init__(self, cost: float = 0, action_probs: List[ActionScore] = []):
+        self.cost = cost
+        self.action_probs = action_probs
+
+    @staticmethod
+    def from_example(ex: "Example"):
+        cost = ex.get_ccb_cost()
+        action_probs = []
+        for i in range(ex.get_ccb_num_included_actions()):
```

Comment:
`explicit_included_actions` does not tell you how many fields there are in the `probabilities` list. You need a function to get length of `probabilities`